taxonomy-editor-frontend/src/pages/editentry/ListAllEntryProperties.jsx

```diff
@@ -0,0 +1,21 @@
+import { Typography } from "@mui/material";
+import ListAllProperties from "./ListAllProperties";
+import ListTranslations from "./ListTranslations";
+
+// Parent component used for rendering sub-components
+// Sub-components are used to render info for an "entry"
+
+const ListAllEntryProperties = ({ nodeObject, setNodeObject }) => {
+    
+    let languageNames = new Intl.DisplayNames(['en'], {type: 'language'});
```

Comment:
maybe pass use this directly in `ListTranslations`, also no need for it to be a `let`